wiki/Beatmap_Spotlights/ru.md

```diff
@@ -0,0 +1,152 @@
+---
+tags:
+  - charts
+  - Ranking Charts
+  - Seasonal Spotlights
+  - спотлайт
+  - спотлайты
+  - чарт
+  - чарты
+  - ранкинг-чарты
+---
+
+# Beatmap Spotlights
+
+![Логотип Beatmap Spotlights](/wiki/shared/news/banners/beatmap-spotlights.jpg)
+
+**Beatmap Spotlights** (*Spotlights*, жарг. *спотлайт*, в прошлом — ранкинг-чарты) — проект, участники которого регулярно выпускают подборки интересных или значимых [карт](/wiki/Beatmap), выделяющихся своим содержанием или оформлением. После выхода очередной подборки открывается [сезонная лига](#сезоны), где игроки могут соревноваться за право получения [значков для профиля](#призы).
+
+Текущий сезон — зима 2023.
+
+На странице группы [кураторов Beatmap Spotlight](https://osu.ppy.sh/groups/48) приведён полный список её участников. Чтобы узнать об их обязанностях, см. [соответствующую статью](/wiki/People/The_Team/Beatmap_Spotlight_Curators).
+
+## Участие
+
+Чтобы состязаться с другими игроками, скачайте последнюю версию клиента osu!(lazer) со [страницы релизов на GitHub](https://github.com/ppy/osu/releases/latest).
```

Comment:
It makes sense to mention the recently updated client download page on the site (https://osu.ppy.sh/home/download)

Reply:
good call, I'll add it to all pages at once in a separate pull requests